src/modules/config.js

```diff
@@ -12,6 +12,31 @@ Object.values(servicesConfigJson.config).forEach(service => {
     )
 })
 
+const
+    apiURL = process.env.API_URL || '',
+
+    // WEB mode related environment variables
+    webEnvs = {
+        webPort: process.env.WEB_PORT || 9001,
+        webURL: process.env.WEB_URL || '',
+        showSponsors: !!process.env.SHOW_SPONSORS,
+        isBeta: !!process.env.IS_BETA,
+        plausibleHostname: process.env.PLAUSIBLE_HOSTNAME || '',
+        apiURL
+    },
+
+    // API mode related environment variables
+    apiEnvs = {
+        apiPort: process.env.API_PORT || 9000,
+        apiName: process.env.API_NAME || 'unknown',
+        corsWildcard: process.env.CORS_WILDCARD !== '0',
+        corsURL: process.env.CORS_URL || '',
+        cookiePath: process.env.COOKIE_PATH || '',
```

Comment:
i'd just let these default to `undefined` if not set (meaning remove the `|| ''`)

Reply:
same with plausibleHostname